Examples/src/main/java/GrayscaleImage.java

```diff
@@ -0,0 +1,125 @@
+ *  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ *  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ *  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ *  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ *  SOFTWARE.
+ * </summary>
+ * --------------------------------------------------------------------------------------------------------------------
+ */
+
+import com.aspose.imaging.cloud.sdk.model.requests.CreateGrayscaledImageRequest;
+import com.aspose.imaging.cloud.sdk.model.requests.GrayscaleImageRequest;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+/**
+ * Grayscale image example.
+ */
+public class GrayscaleImage extends ImagingBase {
+    /**
+     * Initializes a new instance of the GrayscaleImage class
+     *
+     * @param imagingApi The imaging API.
+     */
+    public GrayscaleImage(com.aspose.imaging.cloud.sdk.api.ImagingApi imagingApi) {
+        super(imagingApi);
+        printHeader("Grayscale image example:");
+    }
+
+    /**
+     * Gets the name of the example image file.
+     *
+     * @return The name of the example image file.
+     */
+    @Override
+    protected String getSampleImageFileName() {
+        return "GrayscaleSampleImage.bmp";
+    }
+
+    /**
+     * Grayscale the image from cloud storage.
+     *
+     * @throws Exception
+     */
+    public void GrayscaleImageFromStorage() throws Exception {
+        System.out.println("Grayscales the image from cloud storage");
+
+        uploadSampleImageToCloud();
+
+        String folder = CloudPath; // Input file is saved at the Examples folder in the storage
+        String storage = null; // We are using default Cloud Storage
+
+        GrayscaleImageRequest request = new GrayscaleImageRequest(getSampleImageFileName(), folder, storage);
+
+        System.out.println(String.format("Call GrayscaleImage"));
+
+        byte[] updatedImage = ImagingApi.grayscaleImage(request);
+        saveUpdatedSampleImageToOutput(updatedImage, false, "bmp");
+
+        System.out.println();
+    }
+
+    /**
+     * Grayscale an existing image, and upload updated image to Cloud Storage.
+     *
+     * @throws Exception
+     */
+    public void GrayscaleImageAndUploadToStorage() throws Exception {
+        System.out.println("Grayscales the image and upload to cloud storage");
+
+        uploadSampleImageToCloud();
+
+        String folder = CloudPath; // Input file is saved at the Examples folder in the storage
+        String storage = null; // We are using default Cloud Storage
+
+        GrayscaleImageRequest request = new GrayscaleImageRequest(getSampleImageFileName(), folder, storage);
+
+        System.out.println(String.format("Call GrayscaleImage"));
+
+        byte[] updatedImage = ImagingApi.grayscaleImage(request);
+        uploadImageToCloud(getModifiedSampleImageFileName(false, "bmp"), updatedImage);
+
```

Comment:
bmp is Windows burn format. Would you mind to use  in the case more universal formats aka jpeg or png ?

Reply:
> bmp is Windows burn format. Would you mind to use in the case more universal formats aka jpeg or png ?

What do you mean "a burn format"? All my examples of the grayscale feature use bmp images. Changing and testing them will require some time, is that an urgent task? Can it be postponed until the next release?